libraries/chain/controller.cpp

```diff
@@ -4007,8 +4024,8 @@ void controller::get_finalizer_state( finalizer_state& fs ) const {
 }
 
 // called from net threads
-void controller::notify_hs_message( const uint32_t connection_id, const hs_message& msg ) {
-   my->pacemaker->on_hs_msg(connection_id, msg);
+void controller::process_vote_message( const hs_vote_message& vote ) {
+   my->block_data.aggregate_vote(vote);
```

Comment:
If this returns `true` we need to call the bcast function of net_plugin.

Reply:
I asked at one of the meetings, @arhag said we just needed to aggregate the votes.

Reply:
Do we have an issue for it? We need that for the test net as vote messages will not propagate without it.

Reply:
Will double check with him at tomorrow's meeting.

Reply:
Will broadcast.